### DISCUSSION TITLE
docker下master的fe编译UI时一直异常

### DISCUSSION THREAD

**Discussion by feelshana** [apache/doris#29168] [A - General / Q&A]
在docker下master
--fe 
编译到UI这一步时，一直异常
enoent ENOENT: no such file or directory, rename '/root/doris/ui/node_modules/.ansi-escapes.DELETE/node_modules/type-fest' -> '/root/doris/ui/node_modules/ansi-escapes/node_modules/type-fest'
npm ERR! enoent This is related to npm not being able to find a file.

![image](https://github.com/apache/doris/assets/10239391/9a5949fd-fbf1-4382-9cf9-77ff0d22e0e4)

**Comment by LemonLiTree** [comment]
推荐ldb

**Reply by feelshana** [reply]
是用的apache/doris:build-env-ldb-toolchain-latest镜像 拉的master主干代码

**Reply by LemonLiTree** [reply]
https://doris.apache.org/docs/install/source-install/compilation-with-ldb-toolchain/
